tests/behaviour/expectations/behaviour.ts

```diff
@@ -986,6 +986,20 @@ export const expectations = flatten(
           Expect.Simple('test', [], []),
           Expect.Simple('returnTest', [], ['12', '0']),
         ]),
+        new Dir('modifiers', [
+          File.Simple('modifier', [
+            Expect.Simple('f', ['90000', '0'], ['10000', '0']),
+            Expect.Simple('f', ['110000', '0'], ['0', '0']),
+          ]),
+          File.Simple('multipleModifiers', [
+            new Expect('modifier', [
+              ['openEvent', [], [], '0'],
+              ['donate', ['238', '0'], ['238', '0'], '0'],
+              ['donate', ['100', '0'], ['338', '0'], '0'],
+              ['closeEvent', [], [], '0'],
+            ]),
```

Comment:
I'd like to see at least one test showing that each modifier is applied here. For example, one where donate is called after closeEvent to show that `isOpen` is applied correctly, and one with value < 100 to show that `cost` is applied correctly